doc/symfony-cache-configuration.rst

```diff
@@ -143,6 +146,79 @@ one of ``client_ips`` or ``client_matcher``*.
 
   **default**: ``PURGE``
 
+
+Purge tags (cache invalidation using tags)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. warning::
+
+    You need at least Symfony 3.4 to use this feature!
+
+
+Symfony's `HttpCache` does not support tags based cache invalidation by default.
+However, this library ships with a `TaggableStore` and a corresponding
+`PurgeTagsListener` which provide this functionality.
+
+Purging tags is only allowed from the same machine by default.
+You can configure the listener just the same as the `PurgeListener` plus the
+`purge_tags_method` and `purge_tags_header`:
+
+* **client_ips**: String with IP or array of IPs that are allowed to
+  purge the cache.
+
+  **default**: ``127.0.0.1``
+
+* **client_matcher**: RequestMatcherInterface that only matches requests that are
+  allowed to purge.
+
+  **default**: ``null``
+
+* **purge_tags_method**: HTTP Method used with purge tags requests.
+
+  **default**: ``PURGETAGS``
+
+* **purge_tags_header**: HTTP Header that contains the comma-separated tags to purge.
+
+  **default**: ``X-Cache-Tags``
```

Comment:
lets move this list below the code example. we should provide an example where to set the options. 

for the formatting, do we have nothing similar in the doc yet? if not, we could use something like we do for configuration options in the bundle doc: https://raw.githubusercontent.com/FriendsOfSymfony/FOSHttpCacheBundle/4fc92af83dd63bb4a2b586336711c8cf9b5af7f9/Resources/doc/reference/configuration/match.rst

Reply:
okay, fair enough. i will go over this after merging and try to find a better formatting then, that better links names and default values